src/lib/tooltip/tooltip.ts

```diff
@@ -273,7 +273,7 @@ export class MdTooltip implements OnDestroy {
 
   /** Handles the keydown events on the host element. */
   _handleKeydown(e: KeyboardEvent) {
-    if (this._tooltipInstance!.isVisible() && e.keyCode === ESCAPE) {
+    if (this._tooltipInstance && this._tooltipInstance.isVisible() && e.keyCode === ESCAPE) {
```

Comment:
Sorry for the unsolicited comments. I just ran into the issue in our application. You can replace this is idiom with `this._isTooltipVisible()`.